src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ViewPenaltiesController.java

```diff
@@ -84,15 +87,17 @@ public String getViewPenalties(@PathVariable String companyNumber,
             return ERROR_VIEW;
         }
 
-        model.addAttribute("outstanding", lateFilingPenalty.getOutstanding());
-        model.addAttribute("madeUpDate",
-                LocalDate.parse(lateFilingPenalty.getMadeUpDate(),
-                        DateTimeFormatter.ofPattern("uuuu-MM-dd", Locale.UK))
-                        .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK)));
+        DecimalFormat formatter = new DecimalFormat("#,###");
+        String formattedOutstanding = formatter.format(lateFilingPenalty.getOutstanding());
```

Comment:
Can you pull this out into a method in the 'PenaltyUtils' class? Hoping this stuff gets into a service layer at some stage but for now if we can not add too much more to the controllers that would be great. It will also make that method much easier to unit test.